README.txt

```diff
@@ -1,6 +1,19 @@
 XStatic-Font-Awesome
 --------------------
 
+LICENSE
+-------
+
+Font Awesome Free is free, open source, and GPL friendly. You can use it for commercial projects, open source projects, or really almost whatever you want.
+
+- Icons — CC BY 4.0 License
+  - In the Font Awesome Free download, the CC BY 4.0 license applies to all icons packaged as .svg and .js files types.
+- Fonts — SIL OFL 1.1 License
+  - In the Font Awesome Free download, the SIL OLF license applies to all icons packaged as web and desktop font files.
+- Code — MIT License
+  - In the Font Awesome Free download, the MIT license applies to all non-font and non-icon files.
+
+
```

Comment:
How about moving this to the bottom of the file (if this is needed at all) and just say:

License: See xstatic/pkg/font_awesome/data/LICENSE.txt

Reply:
yep so true, I use your suggestion.